CustomApps/lyrics-plus/index.js

```diff
@@ -48,7 +48,7 @@ const CONFIG = {
 		["font-size"]: localStorage.getItem("lyrics-plus:visual:font-size") || "32",
 		["translation-mode:japanese"]: localStorage.getItem("lyrics-plus:visual:translation-mode:japanese") || "furigana",
 		["translation-mode:chinese"]: localStorage.getItem("lyrics-plus:visual:translation-mode:chinese") || "cn",
-		["translate"]: getConfig("lyrics-plus:visual:translate"),
+		["translate"]: localStorage.getItem("lyrics-plus:visual:translate") || false,
```

Comment:
```suggestion
		["translate"]: getConfig("lyrics-plus:visual:translate", false),
```
![image](https://user-images.githubusercontent.com/77577746/227769210-63768f4f-4958-4689-b187-c39c94e152b8.png)
`getConfig` returns a boolean, `localStorage.getItem` returns a string, which evaluates to `true` on `"false"`